### ISSUE TITLE
`--text` option missing in 1.38

### ISSUE THREAD

**Issue by rodrigoprimo** [semgrep/semgrep#8610]
**Describe the bug**
semgrep 1.38 is failing for us with the following error message indicating that the `--text` doesn't exist anymore:

```
semgrep --error --text --metrics=off -c "/src/tools/wpscan-semgrep-rules/audit" lib/
semgrep scan: unknown option '--text', did you mean '--test'?
Usage: semgrep scan [OPTION]… [TARGETS]…
Try 'semgrep scan --help' for more information.
exiting with error status 2: osemgrep --error --text --metrics=off -c /src/tools/wpscan-semgrep-rules/audit lib/
```

The same command worked fine on 1.37. I'm not sure if this was an intentional change or a bug. If it was an intentional change, is there an alternative?

**To Reproduce**

- Run the command below on 1.38 and see the error:

```
semgrep --error --text --metrics=off -c "/src/tools/wpscan-semgrep-rules/audit" lib/
```

**Expected behavior**
The command above works without an error as it does on 1.37.

**What is the priority of the bug to you?**

- [ ] P0: blocking your adoption of Semgrep or workflow
- [X] P1: important to fix or quite annoying
- [ ] P2: regular bug that should get fixed

**Environment**
If not using semgrep.dev: are you running off docker, an official binary, a local build?

Docker

**Use case**
Pinned our CI to 1.37 but will be stuck there will this issue is not fixed.

**Comment by aryx** [comment]
Not intentional. Will fix ASAP.

**Comment by rodrigoprimo** [comment]
Thanks

**Comment by aryx** [comment]
@rodrigoprimo should be fixed in 1.38.2 released just now
